tock/tock/templates/base.html

```diff
@@ -62,9 +62,15 @@
       {% block js %}
       {% endblock js %}
 </div>
-  <footer>
-    <p><i>The current release is <code>{{ x_tock_release_name }}</code>, <a href="{{ x_tock_release_url }}" title="{{ x_tock_release_name }}">view this release on GitHub</a></i>.</p>
-  </footer>
-
+    <footer>
+       <div class="usa-footer-secondary_section">
+        <div class="usa-grid">
+          <div class="usa-footer-logo">
+            <p><i>The current release is <code>{{ x_tock_release_name }}</code>, <a href="{{ x_tock_release_url }}" title="{{ x_tock_release_name }}">view this release on GitHub</a></i>.</p>
```

Comment:
Do we need three wrappers (four counting the original footer element) just to put some side margin on a paragraph? I get that we're reusing some prior art here, and I appreciate that, but is there a lighter-weight alternative?

Reply:
I'm taking from the USWDS. Feel free to edit a wrapper out if that's better!